src/FsAutoComplete/LspHelpers.fs

```diff
@@ -428,7 +430,8 @@ module Structure =
       StartLine = lsp.Start.Line
       EndCharacter = Some lsp.End.Character
       EndLine = lsp.End.Line
-      Kind = kind }
+      Kind = kind
+      CollapsedText = None }
```

Comment:
We could potentially take the first line of text to use as the folded range here?

Reply:
I think it's fine to let the client decide. Not sure when we would want to actually specify the text when it's folded.